Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -91,32 +101,75 @@ void HiresTexture::Update()
 
   const std::string& game_id = SConfig::GetInstance().GetGameID();
   const std::string texture_directory = GetTextureDirectory(game_id);
-  std::vector<std::string> extensions{
-      ".png", ".bmp", ".tga", ".dds",
-      ".jpg"  // Why not? Could be useful for large photo-like textures
-  };
+  std::vector<std::string> extensions{".dds", ".png", ".bmp",  ".tga",  ".tgb",
+                                      ".jpg", ".jpe", ".jpeg", ".jfif", ".hdr"};
```

Comment:
We want to drop technical debt (meaning we want to drop SOIL), not keep it just because it might be useful for someone. Otherwise we would still be having things like a Direct3D 9 backend...